### DISCUSSION TITLE
An error occured

### DISCUSSION THREAD

**Discussion by muhshif** [btschwertfeger/python-cmethods#31] [Q&A]
I was actually testing this "CMethods" and met with this following error:

AttributeError: module 'cmethods.CMethods' has no attribute 'adjust_3d'

I have installed "CMethods", I used the 1D time series data, etc. 
But, still I am not sure why this is happening?

**Comment by btschwertfeger** [comment]
Hey @muhshif, which version of python-cmethods do you use?

I think that you are using a version less than v1.0.0. 

The error says that "module [...]" so the module indeed do not has the attribute. In version<v1.0.0 - the CMethods class of the module CMethods has this attribute. 

So in case you are using pyhton-cmethods<v1.0.0 you can access the function like this:
```python
from cmethods.CMethods import CMethods
...
CMethods.adjust_3d(...)
``` 

I recommend upgrading to the latest version (v1.0.0) - or wait until v1.0.1 that I will publish today or tomorrow. Here the CMethods class can be imported and used like: 

```python
from cmethods import CMethods as cm
... 
cm.adjust_3d(....)
```
